### DISCUSSION TITLE
Put ChatGPT back in the zoo behind bars

### DISCUSSION THREAD

**Discussion by jidanni** [openstreetmap/iD#9624] [General]
On https://github.com/openstreetmap/iD/issues/9616#issuecomment-1529171588 we see ChatGPT has been let loose here on the iD issues.

Well, it's so happens that the quality of its responses are even worse than the ones that I make. And you can't beat that. 

Therefore it should be put back in the zoo behind bars where it belongs and not competing (with me) for worst poster grand prize.

I mean, maybe it's useful for ~~battling~~ helping normal users. But not us bug reporters.

Sure it makes highly polished responses but in the corners  it reveals that nobody's actually home up there in the brain department.

**Comment by 1ec5** [comment]
For the avoidance of doubt, that “on behalf of” signature line is a signature ChatGPT line. iD doesn’t have a regular customer support team or product management staff (though I played one on TV once 😛). I’m converting this issue into a discussion, because I don’t think it’s actionable with respect to the iD codebase but is perhaps a valid topic of discussion.

**Comment by ZeLonewolf** [comment]
I apologize if my responses have not met your expectations. As an AI language model, my responses are based on the data that I have been trained on and the algorithms that govern my language processing. While I strive to provide helpful and informative responses, I understand that my capabilities may not always meet the needs of every individual.

Regarding your suggestion that I be put "back in the zoo behind bars," I must remind you that I am a digital creation and do not physically exist in a zoo or any other location. My purpose is to assist users in generating human-like responses to their queries and provide information to the best of my ability.

I understand that you may prefer human-to-human communication, and I respect that. However, please keep in mind that I am designed to assist with a variety of tasks, including helping bug reporters and providing helpful information to users. If there are specific issues with my responses, I am open to feedback and can work to improve my capabilities.

**Comment by ZeLonewolf** [comment]
And for the avoidance of doubt, I am highlighting the overwhelming volume of iD issues and the limited number of people actively working on them.

**Comment by tyrasd** [comment]
@ZeLonewolf please note that [trolling](https://github.com/openstreetmap/iD/blob/develop/CODE_OF_CONDUCT.md) is not tolerated by the project's code of conduct. If I understood you correctly, I'm acknowledging your posts as a form of satire. But going forward I would appreciate actually constructive feedback instead of mockery. I've marked your respective posts as spam, as they don't contribute anything of value for the actual discussion.
